tests/test_cli_change_lb.py

```diff
@@ -78,6 +79,8 @@ def test_change_namespace_lb_group(caplog, two_gateways):
     caplog.clear()
     cli(["--server-port", "5501", "namespace", "change_load_balancing_group", "--subsystem", subsystem, "--nsid", "1", "--load-balancing-group", anagrpid2])
     assert f"Changing load balancing group of namespace 1 in {subsystem} to {anagrpid2}: Successful" in caplog.text
+    time.sleep(15)
+    assert f"Received request to delete bdev" not in caplog.text
     caplog.clear()
```

Comment:
@gbregman could we add a test  condition that the second gw sees the lb group change?

Reply:
I added checks that we get the creation of the namespace with the new value in the update() of the second gateway.